Doc/library/asyncio-eventloop.rst

```diff
@@ -232,7 +232,7 @@ Scheduling callbacks
    Callbacks are called in the order in which they are registered.
    Each callback will be called exactly once.
 
-   The optional keyword-only *context* argument specifies a
+   The optional keyword argument *context* specifies a
```

Comment:
Why remove "only"? I don't see anything wrong with calling these "keyword-only".

Reply:
IMO, the `-only` is redundant, but it does make things clearer.

Reply:
I agree with Jelle's 2nd comment that `-only` can be left removed.